astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,77 @@
+* Improve security: authentication services can help you to improve the security of your applications by using best practices and the latest security technologies keeping up to date with the latest security patches.
+* Reduce the risk of compliance violations: authentication services can help you to meet compliance requirements by providing features such as two-factor authentication and user provisioning.
+
+The last thing that I want is a call on Saturday night that someone hacked our password database that we built in house two years ago and didn't maintain securely!  
+
+![Meme you can't leak passwords if you don't store passwords, man pointing at head](/img/blogs/i-just-want-authentication-to-work/meme-leak.png)
+
+## Choosing an authentication service
+
+When choosing an authentication service, there are a few factors to consider:
+
+* Features: Consider the features that are important to you, such as two-factor authentication, user provisioning, and support for different authentication methods.
+* Pricing: authentication services can vary in price. Compare the pricing of different services to find the best deal for your needs.
+* Ease of use: Consider how easy the authentication service is to use. You want a service that is easy to set up and manage.
+* Hosting: Do you need flexibility to host your own instance of an authentication Provider or have them host.
+* Integration support: Often times you'll want to use an authentication service as a central user data store. You'll want to make sure you can integrate custom applications, off the shelf commercial software and open source solutions with it.  
+
+## For Developers Who Just Want It To Work
+
+If you are a developer who just wants authentication to work without implementing it yourself, there are a few things that you can do to make your life easier:
+
+* Use an authentication service. There are many different authentication services available, both free and paid. Authentication services can save you the hassle of having to implement and maintain your own authentication system.
+* Use a framework or library that provides authentication functionality. There are many different frameworks and libraries available that provide authentication functionality. Using a framework or library can save you the time and effort of having to implement authentication yourself. If you are ready to get started checkout our [FusionAuth Quickstarts](/docs/quickstarts/).
+* Consider that when Better Napster, wait I mean your app starts to grow, you will want an authentication provider that can handle different forms of authentication such as mobile, desktop and web.
+* Develop on a local instance or self contained envioronment that you can host yourself without risking data corruption in production.
+* Look to your favoriate framework provider and see what they are using and use an authentication framework built for that.
```

Comment:
spelling. Can you please run this through a spellchecker?

Reply:
apparently my grammarly extensions wasn't running sorry about that!